src/language-js/utils/index.js

```diff
@@ -779,8 +785,8 @@ function isSimpleCallArgument(node, depth = 2) {
 
   if (isMemberExpression(node)) {
     return (
-      isSimpleCallArgument(node.object, depth) &&
-      isSimpleCallArgument(node.property, depth)
+      isSimpleCallArgument(getChainProp(node, "object"), depth) &&
+      isSimpleCallArgument(getChainProp(node, "property"), depth)
```

Comment:
We can add  `if (node.type === "ChainExpression") { node = node.expression }` to `isSimpleCallArgument` like other functions do.

Reply:
Or add to L801